packages/server/graphql/types/GitLabIntegration.ts

```diff
@@ -144,42 +146,55 @@ const GitLabIntegration = new GraphQLObjectType<any, GQLContext>({
             projectsFullPaths.add(edge?.node?.fullPath)
           }
         })
-        const projectIssues = [] as ProjectIssueConnection[]
-        const errors = [] as Error[]
-        const hasNextPage = true
-
-        const projectsIssuesPromises = Array.from(projectsFullPaths).map((fullPath) =>
-          manager.getProjectIssues({
+        const parsed = args.after && JSON.parse(args.after)
+        const projectsIssuesPromises = Array.from(projectsFullPaths).map((fullPath) => {
+          const after = (parsed && parsed[fullPath]) ?? ''
+          return manager.getProjectIssues({
             ...args,
-            includeSubepics: true,
-            fullPath
+            fullPath,
+            after
           })
-        )
+        })
+        const projectsIssues = [] as ProjectIssueEdge[]
+        const errors = [] as Error[]
+        let hasNextPage = false
+        const cursors = {} as Cursors
         const projectsIssuesResponses = await Promise.all(projectsIssuesPromises)
         for (const res of projectsIssuesResponses) {
           const [projectIssuesData, err] = res
           if (err) {
+            errors.push(err)
             sendToSentry(err, {userId})
             return
           }
-          const edges = projectIssuesData.project?.issues?.edges
+          const project = projectIssuesData.project
+          if (!project?.issues) return
```

Comment:
-1 I think we don't want to `return` from the resolver but rather `continue` the for loop.